tools/analyzer_plugin/pubspec.yaml

```
@@ -13,7 +13,7 @@ dependencies:
   # Upon release, this should be pinned to the over_react version from ../../pubspec.yaml
   # so that it always resolves to the same version of over_react that the user has pulled in,
   # and thus has the same boilerplate parsing code that's running in the builder.
-  over_react: 5.4.5
+  over_react: ^5.4.5
```

We need to keep this pinned (see comment above); would you mind reverting this change?
```suggestion
  over_react: 5.4.5
```